cac-sandbox.yml

```diff
@@ -29,7 +29,1160 @@ jenkins:
             value: DCD-CFTAPPS-SBOX
           - key: AKS_AAT_SUBSCRIPTION_NAME
             value: DCD-CFTAPPS-SBOX
-
+  clouds:
```

Comment:
this file shouldn't be changed until the plugin has been upgraded on sandbox

Reply:
this version of the plugin https://github.com/hmcts/cnp-jenkins-config/blob/CNP-1283-CNPTest/playbook.yml#L148-L150 support cac

Reply:
cac-sandbox will get applied after this is merged, the plugin has to be upgraded before this PR can be merged

Reply:
why the plugin has to be upgraded before this PR?

Reply:
to decouple the release, we may (and do) need to be able to change the config at any time. if the plugin isn't updated first then the config won't be able to be applied

Reply:
commented out 'clouds' section for now